app/src/features/onboarding/components/auth/components/Form/index.tsx

```diff
@@ -204,7 +206,136 @@ export const AuthForm: React.FC<AuthFormProps> = ({
       <RequestPasswordResetForm email={email} setEmail={setEmail} setAuthMode={setAuthMode} toggleModal={toggleModal} />
     );
   }
-  return (
+  return onboardingVariation === "variant3" ? (
+    <>
+      <div className="w-full onboarding-auth-form-container">
+        {authMode === AUTH.ACTION_LABELS.LOG_IN && warningMessage && isAuthWarningBannerVisible && (
+          <AuthWarningBanner
+            warningMessage={warningMessage}
+            onBannerClose={() => setIsAuthWarningBannerVisible(false)}
+          />
+        )}
+        <div className="variant3">
+          <h2 className="onboarding-auth-form-header">
+            {authMode === AUTH.ACTION_LABELS.SIGN_UP ? "Create your account" : "Sign in to your account"}
+          </h2>
+        </div>
+        <RQButton
+          type="default"
+          className="onboarding-google-auth-button"
+          onClick={handleGoogleSignInButtonClick}
```

Comment:
can you just verify that we are tracking this action with an event in some way

Reply:
Yes we are.